src/main/java/com/conveyal/gtfs/loader/PatternReconciliation.java

```diff
@@ -146,7 +166,7 @@ public static List<GenericStop> getGenericStops() {
      * Define the type of update to be applied to stop times. This is done by comparing the size of the original generic
      * stops against the size of the new generic stops.
      */
-    private static ReconcileType getReconcileType(List<String> originalGenericStopIds, List<GenericStop> genericStops) {
+    private ReconcileType getReconcileType(List<String> originalGenericStopIds, List<GenericStop> genericStops) {
```

Comment:
Maybe rename to `getReconciliationOperation` or `getOperation` and rename the type to `ReconciliationOperation`.